vita-makepkg

```diff
@@ -1588,7 +1588,10 @@ if (( !REPKG )); then
 	if (( NOEXTRACT && ! VERIFYSOURCE )); then
 		warning "$(gettext "Using existing %s tree")" "\$srcdir/"
 	else
-		download_sources
+		if ( download_sources ); then
+			msg "$(gettext "Cannot retrieve %s, skipping...")" "$pkgbase"
+			exit 0
```

Comment:
@Rinnegatamante  this would be `exit 1`. isn't it?

Reply:
The point is exactly to make Travis not halt when a single package is unavailable.